### PR TITLE
review <Card /> & it's microinteractions

### PR DESCRIPTION
**Pull request by eugene-matvejev**

### CODE REVIEW

**Comment by eugene-matvejev** [comment]
advantages:
1) more performant website, as computations are delegated to GPU
2) more performant website, as it do not trigger DOM recompute
3) smaller bungle size
4) entire styling is in CSS


bug with 'pseudo' selectors seems to be not a case anymore

**Comment by eugene-matvejev** [comment]
Screenshot 2019-10-08 at 17 08 01
Screenshot 2019-10-08 at 17 08 07

demo of moving mouse in-out

**Comment by anasufana** [comment]
👍Looking good 
will approve once tests are working

File: src/components/cards/card.module.css [hunk 1]
```diff
@@ -14,7 +14,13 @@
   padding: 74px 32px 24px 32px;
   display: flex;
   flex-direction: column;
-  box-shadow: 0 2px 10px 0 rgba(0,0,0,0.15);
+  box-shadow: 0px 2px 10px rgba(0,0,0,0.2);
```
Comment (eugene-matvejev) [inline src/components/cards/card.module.css]: it was inconsistency :)

**Comment by eugene-matvejev** [comment]
@anasufana can you please look at this? :)

**Pull request merged**